linux/flutter/generated_plugins.cmake

```diff
@@ -8,9 +8,6 @@ list(APPEND FLUTTER_PLUGIN_LIST
   window_size
 )
 
-list(APPEND FLUTTER_FFI_PLUGIN_LIST
```

Comment:
The cmake file changes here look like they are due to an older version of flutter being used on your end. Could you upgrade and try again?